model/config.go

```diff
@@ -885,7 +885,7 @@ func (s *ExperimentalSettings) SetDefaults() {
 }
 
 type AnalyticsSettings struct {
-	MaxUsersForStatistics *int `restricted:"true"`
+	MaxUsersForStatistics *int `restricted:"true" permissions:"-"`
```

Comment:
@catalintomai There was a "whitelist" of permission in the POC, but I wasn't sure if they're still necessary. For now I have migrated all of them with the tag value `-`. But if they're not necessary anymore then I'd just as soon remove this option as it grants extra access.

Reply:
Removed the `"-"` tag values as discussed offline.